### PR TITLE
Make github usernames clickable

### PR DESCRIPTION
**Pull request by woo2**

### CODE REVIEW

File: app/views/users/_attr.html.erb [hunk 1]
```diff
@@ -6,7 +6,9 @@
         <% elsif attr == "mail" %>
             <div class="attr"><a href="mailto:<%= value[0] %>"><%= value[0] %></a></div>
         <% elsif attr == "mobile" %>
-            <div class="attr"><a href="tel:<%= value[0].tr("() -", "") %>"><%= value[0] %></a></div>
+	    <div class="attr"><a href="tel:<%= value[0].tr("() -", "") %>"><%= value[0] %></a></div>
+	<% elsif attr == "github" %>
+            <div class="attr"><a href="https://github.com/:<%= value[0].tr("() -", "") %>"><%= value[0] %></a></div>
```
Comment (JDrit) [inline app/views/users/_attr.html.erb]: I do not think that there should be a `:` inside the URL. There `:` is a HTML thing and not a ruby thing.

**Comment by JDrit** [comment]
Does the `_attr_edit.html.erb` file also have to be edited?

File: app/views/users/_attr.html.erb (hunk 1 above)
Comment (woo2) [inline app/views/users/_attr.html.erb]: You're right. I'll fix that.

**Comment by woo2** [comment]
No, this only changes how user names are displayed.

**Pull request merged**